src/scene_battle_rpg2k.cpp

```diff
@@ -862,19 +862,29 @@ void Scene_Battle_Rpg2k::SelectNextActor() {
 			break;
 	}
 
-	if (random_target || auto_battle || active_actor->GetAutoBattle()) {
-		if (!random_target) {
-			random_target = Main_Data::game_enemyparty->GetRandomActiveBattler();
-		}
-
-		// ToDo: Auto battle logic is dumb
```

Comment:
What I meant with "logic is dumb" is that the RPG_RT auto battle can also invoke skills.
Though I'm not a fan of auto battle wasting magic.

Reply:
For the auto battle command it's a matter of debate. For actors with auto battle set, they should use skills.